--- dnf-2.7.5/dnf/util.py~	2017-10-18 18:48:46.000000000 +0000
+++ dnf-2.7.5/dnf/util.py	2018-07-03 00:24:46.414245536 +0000
@@ -39,6 +39,7 @@
 import sys
 import tempfile
 import time
+import errno
 
 logger = logging.getLogger('dnf')
 
@@ -120,7 +121,7 @@
     try:
         os.makedirs(dname, mode=0o755)
     except OSError as e:
-        if e.errno != os.errno.EEXIST or not os.path.isdir(dname):
+        if e.errno != errno.EEXIST or not os.path.isdir(dname):
             raise e
 
 def empty(iterable):
